src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/Config/AddAzureRmVmssNetworkInterfaceConfigurationCommand.cs

```diff
@@ -69,6 +69,11 @@ public partial class AddAzureRmVmssNetworkInterfaceConfigurationCommand : Micros
             ValueFromPipelineByPropertyName = true)]
         public SwitchParameter EnableAcceleratedNetworking { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true)]
+        public SwitchParameter EnableIPForwarding { get; set; }
```

Comment:
@hyonholee side note for the generator: switch parameters shouldn't have the `ValueFromPipelineByPropertyName` property